--- gitlab-shell-1.9.5/config.yml.example.orig	2014-06-02 04:45:01.891389227 +0200
+++ gitlab-shell-1.9.5/config.yml.example	2014-06-02 04:44:18.214779888 +0200
@@ -1,5 +1,5 @@
 # GitLab user. git by default
-user: git
+user: gitlab
 
 # Url to gitlab instance. Used for api calls. Should end with a slash.
 gitlab_url: "http://localhost/"
@@ -15,10 +15,10 @@
 # Give the canonicalized absolute pathname,
 # REPOS_PATH MUST NOT CONTAIN ANY SYMLINK!!!
 # Check twice that none of the components is a symlink, including "/home".
-repos_path: "/home/git/repositories"
+repos_path: "/var/lib/gitlab/repositories"
 
 # File used as authorized_keys for gitlab user
-auth_file: "/home/git/.ssh/authorized_keys"
+auth_file: "/var/lib/gitlab/.ssh/authorized_keys"
 
 # Redis settings used for pushing commit notices to gitlab
 redis:
@@ -30,7 +30,7 @@
 
 # Log file.
 # Default is gitlab-shell.log in the root directory.
-# log_file: "/home/git/gitlab-shell/gitlab-shell.log"
+log_file: "/var/lib/gitlab/log/gitlab-shell.log"
 
 # Log level. INFO by default
 log_level: INFO
